README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Project Summary
 
-> Describe the problem or mission statement of the project in 2 - 3 sentences.
+this is a web site which can gives you diet depends on your body type by recieving some information from client. there are some sections for articles related to nutrition and toturial for makking healthy foods and also you can purchase some products related to diet
```

Comment:
This is a web app that provides information on dietary changes and health products depending on your body type by receiving personal input from client. There are some sections for articles related to nutrition and cooking and exercise tutorials.